make-release.sh

```diff
@@ -109,8 +109,10 @@ fi
 set -e
 
 # change version/version.go file
-sed -i version/version.go -r -e 's#(Version = ")(v?[0-9.]+)(")#\1'"${VERSION}"'\3#g'
-
+sed -i version/version.go -r -e "s#(Version = \")(v?[0-9.]+)(\")#\1${VERSION}\3#g"
+# change image tag in Makefile
+sed -i Makefile -r -e "s#export DWO_IMG \?= quay.io/devfile/devworkspace-controller:.*#export DWO_IMG \?= quay.io/devfile/devworkspace-controller:${VERSION}#g"
```

Comment:
that works fine but I would say it's double work. Instead maybe move it below and do
```suggestion
sed -i Makefile -r -e "s#export DWO_IMG \?= .*#export DWO_IMG \?= ${DWO_QUAY_IMG}#g"
```
and the same for PROJECT_CLONE

Reply:
ok, thanks, I made an even more generic sed, to just match the image name anywhere in Makefile